README.md

```diff
@@ -3,12 +3,24 @@ Discord APT repository
 
 As has been [repeatedly asked for](https://support.discord.com/hc/en-us/community/posts/360031737491-Give-us-an-apt-repository-Linux-), this is an APT repository for Discord. Note that I do not in anyway claim ownership over the .deb files here, this is just merely a nicer packaging option. Discord: if you want to do this yourself, please tell me about it and I'll point people there instead!
 
-Usage instructions
+## Usage instructions
 --
-1. Create a file `/etc/apt/sources.list.d/discord.list` with the contents `deb https://palfrey.github.io/discord-apt/debian/ ./`
-2. Download the file https://palfrey.github.io/discord-apt/discord-apt.gpg.asc to `/etc/apt/trusted.gpg.d`
-3. `sudo apt-get update`
-4. `sudo apt-get install discord`
+1. Import discord-apt signing key:
+##
```

Comment:
Do not use empty `##` tags. Its for level 2 headers.

You can format code block with triple backticks and `bash` language instead using indent. [How it works](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting)